kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/BatchRestApi.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.client;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.google.gson.Gson;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.kyuubi.KyuubiException;
+import org.apache.kyuubi.client.api.v1.dto.Batch;
+import org.apache.kyuubi.client.api.v1.dto.BatchRequest;
+import org.apache.kyuubi.client.api.v1.dto.GetBatchesResponse;
+import org.apache.kyuubi.client.api.v1.dto.OperationLog;
+
+public class BatchRestApi {
+
+  private KyuubiRestClient client;
+
+  public BatchRestApi(KyuubiRestClient client) {
+    this.client = client;
+  }
+
+  public RestClient getClient() {
+    return this.client.getHttpClient();
+  }
+
+  public Batch createBatch(BatchRequest request) throws KyuubiException {
+    String jsonBody = new Gson().toJson(request);
```

Comment:
Is it possible that, we can use the api in `jackson-databind` to do it.
```
ObjectMapper
```
So that we can remove Gson dependency. It is better to have less dependencies.